models/general/knowledge/en/countries_information.txt

```diff
@@ -10,7 +10,7 @@ eol
 
 #Tells us about time zone of different countries
 
-Time Zone of *|What is time zone of *| * Time zone 
+Time Zone of *|What is time zone of *| * Time zone|tell me the time zone of * 
```

Comment:
Add 
Do you know question here also and also if you write anything with article i.e tell me the time one of * then we should also make a query without "the" also